### PR TITLE
Add gather step to snc e2e

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: ci-operator/step-registry/gather/snc/gather-snc-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+ref:
+  as: gather-snc
+  from: snc
+  commands: gather-snc-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
```
Comment (stevekuznetsov) [inline ci-operator/step-registry/gather/snc/gather-snc-ref.yaml:9]: These are intended to expose parameters to the users that execute this step. Which of these are parameters that should be set by the user at runtime?

Comment (praveenkumar) [reply ci-operator/step-registry/gather/snc/gather-snc-ref.yaml:9]: @stevekuznetsov as of now, these parameter make use of default value so in the runtime user doesn't need to set anything but in future if any of parameter need to pass by step then these can be used.

**Pull request merged**